nistats/tests/test_dmtx.py

```diff
@@ -293,7 +302,7 @@ def test_design_matrix10():
     X, names = design_matrix_light(frame_times, paradigm, hrf_model=hrf_model,
                          drift_model='polynomial', drift_order=3,
                          fir_delays=range(1, 5))
-    onset = paradigm.onset[paradigm.con_id == 'c0'].astype(np.int)
+    onset = paradigm['onset'][paradigm['name'] == 'c0'].astype(np.int)
```

Comment:
Minor remark: you can use the dot notation to access columns in pandas (as long as your column name is a valid python identifier IIRC). So I reckon this should work:

```python
paradigm.onset[paradigm.name == 'c0']
```